contracts/erc20-gateway/ERC20Gateway.sol

```diff
@@ -0,0 +1,75 @@
+pragma solidity >=0.5.0 <0.6.0;
+
+// Copyright 2020 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import "../proxies/MasterCopyNonUpgradable.sol";
+import "../message-bus/MessageBus.sol";
+
+contract ERC20Gateway is MasterCopyNonUpgradable, MessageBus {
```

Comment:
```suggestion
contract ERC20Gateway is MasterCopyNonUpgradable, MessageBus, ERC20GatewayBase {
```

Now we have the `erc20-gateway` folder so we can also move the `ERC20GatewayBase` contract to this folder. Please use `git mv` command only

Reply:
Done!